sdk/iOS/src/MSQueryResult.h

```diff
@@ -10,7 +10,7 @@
 ///@{
 
 /// The result from a successful read
-@property (nonatomic, strong, readonly)           NSArray *items;
+@property (nonatomic, strong, readonly, nonnull)           NSArray<NSDictionary *> *items;
```

Comment:
There is an example of it being constructed from a `MSSyncContextReadResult`. The `MSSyncContextReadResult` has nonnull items property, so I propagated that through to this.

Haven't found a pathway where it is constructed with a nil property